mu-plugins/blocks/global-header-footer/header.php

```diff
@@ -114,7 +114,7 @@ class="wp-block-search__button-outside wp-block-search__text-button global-heade
 						class="wp-block-search__input"
 						name="search"
 						value=""
-						placeholder="<?php echo esc_attr_x( 'Search WordPress.org...', 'input field placeholder', 'wporg' ); ?>"
+						placeholder="<?php echo esc_attr_x( 'Search WP.org...', 'input field placeholder', 'wporg' ); ?>"
```

Comment:
@beafialho , is this to shorten the length of the placeholder, or something else?

It seems incongruent from a user perspective, since only insiders are familiar with that domain alias.

Reply:
@iandunn it is. I don't feel strongly about it though. We can leave `WordPress.org` as placeholder text.